### PR TITLE
Update CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by mariaokounkova**
Remove the template code of conduct, and replace with the SXS code of conduct. No new changes to the CoC were introduced, and all of the section formatting was kept as in the original document.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,76 +1,120 @@
...
+
+We have adopted the same policy as the LIGO Scientific Collaboration Ombudsperson; see this document, replacing LSC with SXS Collaboration, and LSC Spokesperson with the SXS Executive Committee, for more details about the [Ombudspersons role](https://dcc.ligo.org/public/0099/M1300006/001/LSCOmbudsperson.pdf).
+
+
+## Advocate
+
+The SXS Advocate must be elected by the student and postdoc members of SXS, and be within two years of their Ph.D. (i.e. a senior graduate student or a new postdoc). The SXS executive committee will have a call for nominations and then consult with nominees to see if they are willing to be the advocate. If the SXS executive committee has any concerns with a nominee, they will discuss these concerns with the nominator. The SXS executive committee will then supervise an election among SXS student and postdoc members with ranked-choice voting among those nominees that agreed to stand for election. Once the list of candidates for SXS advocate is announced, the executive committee cannot change the candidate list (unless there is a serious violation of the SXS code of conduct). The Advocate is elected for a one-year term, and may be re-elected.
```
Comment (duetosymmetry) [inline CODE_OF_CONDUCT.md]: These two sections seem to duplicate the content of (new) lines 23-31 above

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,76 +1,120 @@
...
-reported by contacting the project team at support@github.com. All
-complaints will be reviewed and investigated and will result in a response that
-is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
+The Student/Postdoc Advocate (currently Masha Okounkova, email sxs-advocate@black- holes.org) is someone who will advocate positively for the people in the collaboration with less power, specifically high school students, undergraduate students, summer REU students, grad students, and postdocs. It may sometimes be daunting to go to professors, so please feel free to talk to the Advocate about any issues and concerns you may have in the collaboration, especially concerns about culture and environment. The Advocate can bring up (anonymously) any of your concerns with the SXS executive committee if you wish. Other- wise, unless theres a serious issue of safety, the Advocate will keep the Ombudsperson in the loop but otherwise keep your conversation confidential.
```
Comment (duetosymmetry) [inline CODE_OF_CONDUCT.md]: The email address on this line has whitespace between "black-" and "holes.org", so does not get rendered as a clickable link

**Review by duetosymmetry — CHANGES_REQUESTED** [review]
Spotted some formatting issues:

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (duetosymmetry) [reply CODE_OF_CONDUCT.md]: Also, the auto-hyphenation of "Other-wise" has made it into the markdown and should be un-hyphenated here.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,76 +1,120 @@
...
+
+In addition, violations of this code outside these spaces may, in rare cases, affect a persons ability to participate within them, when the conduct amounts to an egregious violation of this code. If you believe someone is violating the code of conduct, we ask that you report it by emailing one or more members of the SXS Executive Committee (see Reporting be- low). If you would rather not formally report your concern, you should feel free to discuss it informally and confidentially with the SXS Ombudsperson.
```
Comment (duetosymmetry) [inline CODE_OF_CONDUCT.md]: "be-low" should have hyphen removed.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (mariaokounkova) [reply CODE_OF_CONDUCT.md]: This is directly copied from the current code of conduct, so to change this duplication we'll have to all discuss and adopt it

Comment (duetosymmetry) [reply CODE_OF_CONDUCT.md]: I actually don't remember where our definitive CoC lives. I looked at [this one](https://github.com/sxs-collaboration/WelcomeToSXS/blob/master/sxs_code_of_conduct.pdf) which agrees with what's on the black-holes.org web site, which must be older, and don't have this duplication.

So, separate issues are:
- Fix the duplication at the origin (where?)
- Update the web site and Welcome repo (linked above) with the new version

**Comment by duetosymmetry** [comment]
LGTM, please squash

**Review by duetosymmetry — APPROVED** [review]

**Pull request merged**